frontend/ReleaseDashboard/TestDashboard.svelte

```diff
@@ -167,10 +167,10 @@
 
     onMount(() => {
         fetchStats();
-
+        let refreshInterval = 300 + 15 + Math.round((Math.random() * 10));
```

Comment:
Why are you using random?
So the refreshes won't happen at the same time?

Reply:
On workspace however the refreshes would be staggered to yeah, prevent them from happening at the same time, as workspace does N requests for N releases and if a person has several tabs open with workspace this could potentially create very high load if they all refresh at the same time.